Description: Add missing ETIME definition for kFreeBSD flavours.
Author: Steven Chamberlain <steven@pyro.eu.org>
Bug-Debian: http://bugs.debian.org/673704
Forwarded: not-yet
Reviewed-by: Didier Raboud <odyx@debian.org>
Last-Update: 2013-03-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libfprint/drivers/uru4000.c
+++ b/libfprint/drivers/uru4000.c
@@ -32,6 +32,10 @@
 
 #include "driver_ids.h"
 
+#ifndef ETIME
+#define ETIME ETIMEDOUT /* For kFreeBSD */
+#endif
+
 #define EP_INTR			(1 | LIBUSB_ENDPOINT_IN)
 #define EP_DATA			(2 | LIBUSB_ENDPOINT_IN)
 #define USB_RQ			0x04
--- a/libfprint/drivers/vfs101.c
+++ b/libfprint/drivers/vfs101.c
@@ -27,6 +27,10 @@
 
 #include "driver_ids.h"
 
+#ifndef ETIME
+#define ETIME ETIMEDOUT /* For kFreeBSD */
+#endif
+
 /* Input-Output usb endpoint */
 #define EP_IN(n)	(n | LIBUSB_ENDPOINT_IN)
 #define EP_OUT(n)	(n | LIBUSB_ENDPOINT_OUT)
--- a/libfprint/drivers/vfs301.c
+++ b/libfprint/drivers/vfs301.c
@@ -21,6 +21,7 @@
 
 #define FP_COMPONENT "vfs301"
 
+
 #include <errno.h>
 #include <signal.h>
 #include <string.h>
@@ -36,6 +37,10 @@
 
 #include "driver_ids.h"
 
+#ifndef ETIME
+#define ETIME ETIMEDOUT /* For kFreeBSD */
+#endif
+
 /************************** GENERIC STUFF *************************************/
 
 /* Callback of asynchronous sleep */
